<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP客户端演示程序</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/ftp.png</normaloff>:/ftp.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelFtpServer">
          <property name="text">
           <string>主机名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditFtpServer">
          <property name="text">
           <string>192.168.245.128</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFtpPort">
          <property name="text">
           <string>端 口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditFtpPort">
          <property name="text">
           <string>21</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFtpUserName">
          <property name="text">
           <string>用户名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditFtpUserName">
          <property name="text">
           <string>anonymous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFtpPwd">
          <property name="text">
           <string>密 码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditFtpPwd">
          <property name="text">
           <string>123456</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pBtnFtpConnect">
        <property name="text">
         <string>连 接</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pBtnFtpCdToParent">
        <property name="text">
         <string>返回上一级目录</string>
        </property>
        <property name="icon">
         <iconset resource="res/res.qrc">
          <normaloff>:/cdtoparent.png</normaloff>:/cdtoparent.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pBtnFtpDown">
        <property name="text">
         <string>下 载</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtnFtpUp">
        <property name="text">
         <string>上 传</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtnExit">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="trWidgetFileList">
      <column>
       <property name="text">
        <string>文件</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>大小</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>拥有者</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>所属组</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>修改日期</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
